CAVEATS.md

```diff
@@ -9,18 +9,22 @@
 ## Implementation Correctness
 
 The following seL4 architectures have platforms with a C-level functional
-correctness proof. Proof support for further platforms within these
-architectures is on the roadmap and expected in 2025.
+correctness proof. Proof support for further platforms for RISC-V is on
+the roadmap and expected in 2026.
 
-- AArch32: Armv7-a with and without hypervisor extensions, no SMMU, with
+- AArch32: Armv7-a with and without hypervisor extensions, no SMMU, no FPU, with
   fast path
-  - Platforms (non-hyp): `sabre` (no FPU), `imx8mm-evk` (with FPU)
-  - Platforms (hyp, no FPU): `tk1`, `exynos5`
-- AArch64: Armv8-a with hypervisor extensions only, no SMMU, with fast path
-  - Platforms: `tx2`, `zynqmp`, `bcm2711` (rpi4)
-- RISC-V: 64-bit only, no fast path
+  - Platforms (non-hyp): `sabre`, `am335x`, `bcm2837`, `exynos4`, `exynos5410`,
```

Comment:
Since there's a lot of platforms verified now, should this be a list instead?

e.g

- Platforms:
    - `sabre`
    - `am335x`
    - ...